.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "assets/Font-Awesome"]
+	path = assets/Font-Awesome
+	url = https://github.com/FortAwesome/Font-Awesome.git
+[submodule "assets/bootstrap-sass"]
+	path = assets/bootstrap-sass
+	url = https://github.com/twbs/bootstrap-sass
```

Comment:
What is "they" ? Not sure what you want to rename... The cache files will be downloaded and cached "correctly" and not changed again. The generated assets will only have the names of the input files so they don't clash in prod/dev mode. This is one of the big changes in the rewrite: Avoiding a bunch of files with different names on every little change. Maybe my blog post can give more input: http://thorsen.pm/perl/2016/02/21/rewriting-assetpack-plugin.html#purging-processed-assets